public/javascripts/SVLabel/src/SVLabel/onboarding/OnboardingStates.js

```diff
@@ -561,15 +613,15 @@ function OnboardingStates (compass, mapService, statusModel, tracker) {
             "panoId": panoId,
             "annotations": null,
             "transition": function () {
-                var completedRate = 14 / numStates;
+                var completedRate = 15 / numStates;
                 statusModel.setMissionCompletionRate(completedRate);
                 statusModel.setProgressBar(completedRate);
                 tracker.push('Onboarding_Transition', {onboardingTransition: "tag-attribute-2"});
```

Comment:
Looks like you forgot to update this

```suggestion
                tracker.push('Onboarding_Transition', {onboardingTransition: "tag-attribute-3"});
```